docs/topics/whatsnew1620.md

```diff
@@ -0,0 +1,851 @@
+>
+{type="note"}
+
+With Kotlin 1.6.20, you can try the Alpha version of the new Kotlin/Native memory manager.
+It eliminates the differences between the JVM and Native platforms to provide a consistent developer experience in multiplatform projects.
+For example, you’ll have a much easier time creating new cross-platform mobile applications that work on both Android and iOS.
+
+The new Kotlin/Native memory manager lifts restrictions on object-sharing between threads.
+It also provides leak-free concurrent programming primitives that are safe and don’t require any special management or annotations.
+
+The new memory manager will become the default in future versions, so we encourage you to try it now.
+Check out our [blog post](https://blog.jetbrains.com/kotlin/2021/08/try-the-new-kotlin-native-memory-manager-development-preview/) to learn more about the new memory manager and explore demo projects, or jump right to the [migration instructions](https://github.com/JetBrains/kotlin/blob/master/kotlin-native/NEW_MM.md) to try it yourself.
+
+Try using the new memory manager on your projects to see how it works and share feedback in our issue tracker, [YouTrack](https://youtrack.jetbrains.com/issue/KT-48525).
+
+### Concurrent implementation for the sweep phase in new memory manager
+
+If you have already switched to our new memory manager, which was [announced in Kotlin 1.6](whatsnew16.md#preview-of-the-new-memory-manager), you might notice a huge execution time improvement: our benchmarks show 35% improvement on average.
+Starting with 1.6.20, there is also a concurrent implementation for the sweep phase available for the new memory manager.
+This should also improve the performance and decrease the duration of garbage collector pauses.
+
+To enable the feature for the new Kotlin/Native memory manager, pass the following compiler option:
+
+```bash
+-Xgc=cms 
+```
+
+Feel free to share your feedback on the new memory manager performance in this [YouTrack issue](https://youtrack.jetbrains.com/issue/KT-48526).
+
+### Instantiation of annotation classes
+
+In Kotlin 1.6.0, instantiation of annotation classes became [Stable](components-stability.md) for Kotlin/JVM and Kotlin/JS.
+The 1.6.20 version delivers support for Kotlin/Native.
+
+Learn more about [instantiation of annotation classes](annotations.md#instantiation).
+
+### Interop with Swift async/await: returning Void instead of KotlinUnit
+
+> Concurrency interoperability with Swift async/await is [Experimental](components-stability.md). It may be dropped or changed at any time.
+> You should use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-47610).
+>
+{type="warning"}
+
+We’ve continued working on the [experimental interop with Swift’s async/await](whatsnew1530.md#experimental-interoperability-with-swift-5-5-async-await) (available since Swift 5.5).
```

Comment:
```suggestion
We've continued working on the [experimental interop with Swift's async/await](whatsnew1530.md#experimental-interoperability-with-swift-5-5-async-await) (available since Swift 5.5).
```